tests/unit/sagemaker/jumpstart/test_utils.py

```diff
@@ -906,3 +914,16 @@ def test_mime_type_enum_from_str():
         for suffix in suffixes:
             mime_type_with_suffix = mime_type + suffix
             assert MIMEType.from_suffixed_type(mime_type_with_suffix) == mime_type
+
+
+def test_stringify_object():
+    class MyTestClass:
+        def __init__(self):
+            self.blah = "blah"
+            self.wtafigo = "eiifccreeeiuclkftdvttufbkhirtvvbhrieclghjiru"
```

Comment:
although the string is long, it's functionally the same test as line 922.

Could you vary the value type, for example could you try `int`, `float`, `List[str]`, `Dict[str, str]` and `List[Dict[str, str]]` ?